src/_posts/platform/app/2000-01-01-procfile.md

```diff
@@ -9,48 +9,150 @@ Procfile support is one of our
 [compatibility layers]({% post_url platform/getting-started/2000-01-01-heroku-compatibility %})
 that makes Scalingo compatible with Heroku.
 
-## Definition
+## About the Procfile
 
-The Procfile is a way to define how the platform will try to start your containers.
-In this file, you can define all the different process you would like to run in your
-project **from the same code base**. It is commonly used to define how to start workers which will consume
-asynchronous jobs.
+The **Procfile** is a simple text file specifying how the platform should start
+the Managed Runtime(s) of your application.
 
-A Procfile is a text file using the YAML format describing how to start
-each type of container for your app. It has to be located at the root of your project.
-Environment variables will be interpolated.
+It allows to declare the commands executed in the Managed Runtime(s) on
+startup.
 
-```yaml
-web: bundle exec rails server -p $PORT -e $RAILS_ENV -b 0.0.0.0
-worker: bundle exec sidekiq -e $RAILS_ENV
+Each of these commands is associated to a **process type**, which is basically
+a name that uniquely identifies the type of process.
+
+You can define as many process types as you need for your application, **from
+the same code base**.
+
+Some common use cases for process types:
+- define a process type called `web` to start a process handling HTTP requests.
+- define a process type called `worker` to handle some asynchronous
+  computation.
+- define a `clock` process type to schedule some recurring jobs.
+- define a `console` process type to boot up [one-offs]({% post_url platform/app/2000-01-01-tasks %})
+  and interact with a database from the CLI.
```

Comment:
Even if technically possible. This is not documented and not really included in the product. I would remove this usecase.

Reply:
Argh, I like this use-case and added it on purpose :D
Anyway, I removed the sentence in https://github.com/Scalingo/documentation/pull/2020/commits/64da5f624d74d705bb2f074587d77ad065c7211c